common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -316,6 +316,16 @@ trait FeatureSwitches {
     exposeClientSide = true
   )
 
+  val DiscussionOrderByOldest = Switch(
+    SwitchGroup.Feature,
+    "discussion-order-by-oldest",
+    "If on, comments default sort ordering will be oldest first. If off, then the default is newest first",
+    owners = Seq(Owner.withGithub("nicl")),
+    safeState = Off,
+    sellByDate = never,
+    exposeClientSide = true
+  )
+
```

Comment:
@jamesgorrie where is a good place to do this? It is intended for testing, but it's difficult to formally A/B test so we were just going to try it and get feedback from Mods.